ion-schema/src/main/kotlin/com/amazon/ionschema/internal/TypeCore.kt

```diff
@@ -64,8 +63,3 @@ internal class TypeCore(
         }
     }
 }
-
-internal fun IonType.schemaTypeName() = when (this) {
-    IonType.DATAGRAM -> "document"
-    else -> this.toString().toLowerCase()
-}
```

Comment:
This function was moved to `IonValueUtils.kt` since I'm also using it in `OrderedElements.kt` now.